library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/SupportFragmentResourceFinder.java

```diff
@@ -65,6 +66,9 @@ public ViewGroup getPromptParentView()
     {
         if (this.parent == null)
         {
+            ViewParent parent = this.fragment.getView().getParent();
+            while (parent.getClass().getName().contains("FragmentContainerView"))
+                parent = parent.getParent();
             //noinspection ConstantConditions
             this.parent = (ViewGroup) this.fragment.getView().getParent();
```

Comment:
Should this be?
```java
this.parent = (ViewGroup) parent;
```

Reply:
yes you are correct, that's what the actual code that worked for me was, I guess I messed up something while commiting the changes